streams/us.m3u

```diff
@@ -1586,5 +1586,21 @@ https://d3qm7vzp07vxse.cloudfront.net/v1/master/77872db67918a151b697b5fbc23151e5
 https://dai.google.com/linear/hls/event/RDmbn03GRCOLNG1eetEKhg/master.m3u8
 #EXTINF:-1 tvg-id="" status="error",AXN East Asia (Indonesian subtitle) (576p)
 http://210.210.155.37/dr9445/h/h141/index.m3u8
-#EXTINF:-1 tvg-id="" status="error",TV4 Bullhead City AZ
+#EXTINF:-1 tvg-id="" status="error",TV4 Bullhead City AZ (360p)
 https://cdn3.wowza.com/5/UWpORHhLSEs5SkJs/bullheadcity/G0860_001/playlist.m3u8
+#EXTINF:-1 tvg-id="KBEV.us" status="",KBEV 6 (1080p)
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="KBEV.us" status="",KBEV 6 (1080p)
```

Website says Spectrum Channel 6. KBEV 6 is a student-run channel from Beverly Hills